robotiq_driver/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ target_include_directories(
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include>
 )
+target_link_libraries(${PROJECT_NAME} atomic)
```

Comment:
You shouldn't need to do this to fix the problem -- this seems like a red flag that something is missing a dependency.